components/authz-service/engine/conformance/conformance_v2_test.go

```diff
@@ -82,18 +82,16 @@ func TestV2IsAuthorized(t *testing.T) {
 
 func TestV2ProjectsAuthorized(t *testing.T) {
 	ctx, engines := setup(t)
-	sub, act, res, proj1, proj2 := "user:local:admin", "iam:users:create", "iam:users", "proj-1", "proj-2"
+	sub, act, res := "user:local:admin", "iam:users:create", "iam:users"
+	proj1, proj2, proj3, proj4, unassigned := "proj-1", "proj-2", "proj-3", "proj-4", "(unassigned)"
```

Comment:
Please use constant so the definition is only given once per service:
s/"(unassigned)"/constants.UnassignedProjectID/